code/modules/mob/living/simple_animal/hostile/bees.dm

```diff
@@ -243,6 +243,12 @@
 		return 1
 	return 0
 
+/mob/living/simple_animal/hostile/poison/bees/attacked_by(obj/item/I, mob/living/user)
+	. = ..()
+	if(istype(I, /obj/item/weapon/melee/flyswatter))
+		user << "<span class='warning'>You easily splat the [src].</span>"
+		new /obj/effect/decal/cleanable/deadcockroach(get_turf(src))
```

Comment:
bees are now cockroaches.

Why not just called the `death` proc for bees instead?